test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -566,9 +566,75 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			By("Saving the old performance profile")
 			initialProfile = profile.DeepCopy()
 		})
+		When("RealTime Workload with RealTime Kernel set to false", func() {
+			It("[test_id:50991][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+				By("Modifying profile")
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
```

Comment:
HihgPowerConsumption to true and Realtime to false is the default when no workloadhints are set in performanceprofile. Bettet not to set anything here and point in the title that is the default case.

Reply:
Is it not the  otherway around. "RealTime: True and Highpowerconsumption: False "

Reply:
Yes its ok, I meant that "RealTime: True and Highpowerconsumption: False " and default, thus, with no workloadHints its the same, just to test the default behaviors. But its not necessary.